spec/kitchen/provisioner/habitat_spec.rb

```diff
@@ -11,11 +11,10 @@
 require "kitchen/verifier/dummy"
 
 def wrap_command(code, left_pad_length = 10)
-  command = "sh -c '\n"
-  command << "TEST_KITCHEN=\"1\"; export TEST_KITCHEN\n"
-  command << "CI=\"true\"; export CI\n" if ENV["CI"]
-  command << code
-  command <<"'"
+  left_padded_code = code.map do |line|
+    line.rjust(line.length + left_pad_length)
+  end.join("\n")
+  command = "#{left_padded_code}\n"
```

Comment:
![good news](https://user-images.githubusercontent.com/517302/81315269-1d38b980-9058-11ea-81ef-380ed9db4626.gif)

I believe this `wrap_command` method can go away now. It currently exists solely to add left-padded whitespace to multiline strings with a number of spaces given. All uses of this method in the tests have the number of spaces given set to `0` because the HEREDOCs are now using the squiggly variant that _removes_ leading whitespace from the original string.

Reply:
I removed this method in my most recent commit. Thanks for the tip on the squigglies. That helped me in some of the tests too.